developer-experience/recipes/bridge-to-kubernetes.md

```diff
@@ -0,0 +1,49 @@
+
+# Bridge to Kubernetes
+
+Bridge to Kubernetes allows developers to seamlessly work on services both in a shared cluster and locally in debug mode.
+
+To learn more about Bridge to Kubernetes:
+
+- [Introduction](https://code.visualstudio.com/docs/containers/bridge-to-kubernetes)
+- [How it works](https://docs.microsoft.com/en-us/visualstudio/containers/overview-bridge-to-kubernetes)
+- [Configuration](https://docs.microsoft.com/en-us/visualstudio/containers/configure-bridge-to-kubernetes)
+- [VS Code Setup](https://docs.microsoft.com/en-us/visualstudio/containers/bridge-to-kubernetes?view=vs-2019)
+- [Repository](https://github.com/Microsoft/mindaro)
+
+## Useful Scenarios
```

Comment:
I would suggest also explaining how this relates with the content explained in the root readme. For example, what I suspect this relates to is helping with ease E2E start/debug; which is one of the essential tasks mentioned below. Also when would you use this approach. I believe you kind of hinted at this in the first line but, may be worth diving a bit deeper into when they might need to adopt this. 

For example, e2e debug requires integration with external services on the same cluster. Rather than run those services locally (which may not even be possible), you can use this to debug services in the cluster where these external services are already running.